### DISCUSSION TITLE
Conda stone-age version

### DISCUSSION THREAD

**Discussion by staniPetrox** [Lightning-AI/pytorch-lightning#15262] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
When I install pl with conda, according to doc with
`conda install pytorch-lightning -c conda-forge`
I will get a pl version with `python -c "import pytorch_lightning as pl;print(pl.__version__)"` and the result is `0.8.5` which is way too low..

What is the problem actually?

**Comment by akihironitta** [comment]
Hi @staniPetrox, thank you for your report. Could you comment on #14374 with the commands you've run?

**Reply by staniPetrox** [reply]
I had the wrong python path
